runtime/src/launch/pbs-gasnetrun_ibv/launch-pbs-gasnetrun_ibv.c

```diff
@@ -170,6 +170,17 @@ static char* chpl_launch_create_command(int argc, char* argv[],
 #else
   mypid = 0;
 #endif
+  sysFilename=(char *)chpl_mem_allocMany((strlen(baseSysFilename) + 
+                                    snprintf(NULL, 0, "%d", (int)mypid) + 1), 
```

Comment:
I had not seen this pattern before—clever.  Given that this same expression appears three times in a row, how about storing it into an `int` and then re-using that int in these three malloc calls.  I.e., something like:

```c
int mypidBuffLen = snprintf(NULL, 0, "%d", (int)mypid);
```